akka-cluster/src/test/scala/akka/cluster/CrossDcHeartbeatSenderSpec.scala

```diff
@@ -36,18 +41,25 @@ class CrossDcHeartbeatSenderSpec extends AkkaSpec("""
   """) with ImplicitSender {
   "CrossDcHeartBeatSender" should {
     "increment heart beat sequence nr" in {
-      val probe = TestProbe()
+
+      val heartbeatProbe = TestProbe()
       Cluster(system).join(Cluster(system).selfMember.address)
       awaitAssert(Cluster(system).selfMember.status == MemberStatus.Up)
-      val underTest = system.actorOf(Props(new TestCrossDcHeartbeatSender(probe)))
+      val underTest = system.actorOf(Props(new TestCrossDcHeartbeatSender(heartbeatProbe)))
+
       underTest ! CurrentClusterState(
         members = SortedSet(
           Cluster(system).selfMember,
           Member(UniqueAddress(Address("akka", system.name), 2L), Set("dc-dc2"), Version.Zero)
             .copy(status = MemberStatus.Up)))
 
-      probe.expectMsgType[Heartbeat].sequenceNr shouldEqual 1
-      probe.expectMsgType[Heartbeat].sequenceNr shouldEqual 2
+      underTest ! ReportStatus()
+      awaitAssert {
+        expectMsgType[CrossDcHeartbeatSender.MonitoringActive]
+      }
```

Comment:
Does the `awaitAssert` still do anything now, since we send only one `ReportStatus`? Perhaps just have `expectMsgType`, potentially/if necessary with a longer timeout?

Reply:
hmm, there is an error. The ReportStatus must be inside. 
The reason is that it can retur MonitoringDormant in the meantime.

Now I remember why I put it inside initially.

Reply:
Fixed. Keeps ask for status until it gets what it needs. I need rest. I did it right and then forget the reason and reverted. 🤦